tests/robot-tests/tests/admin/bau/create_data_block_with_chart.robot

```diff
@@ -391,29 +391,29 @@ Validate line chart embeds correctly
     user checks chart tooltip item contains  ${datablock}  1  Admission Numbers (Nailsea Youngwood): 4,198
 
 Configure basic vertical bar chart
-    [Tags]  HappyPath
+    [Tags]  HappyPath  NotAgainstDev
     user goes to url  ${DATABLOCK_URL}
 
-    user waits until h2 is visible  ${DATABLOCK_NAME}
+    user waits until h2 is visible  ${DATABLOCK_NAME}  120
```

Comment:
I'd recommend as a follow-up piece of work that we assign these wait times to parameters that are configurable e.g. 120 would be ${LONG_WAIT}, 60 would be ${MEDIUM_WAIT} etc.

My driver for this is that if I'm running locally I don't wanna wait 2 minutes to be alerted of a failure, otherwise that'd slow down my own test writing.

Reply:
yeah, that's a good point. Will create a task for this